datalab/datalab.sh

```diff
@@ -38,6 +38,9 @@ if [[ "${ROLE}" == 'Master' ]]; then
     echo 'spark.sql.warehouse.dir=/root/spark-warehouse' >> "${SPARK_CONF}"
   fi
 
+  DATALAB_DIR="${HOME}/datalab"
```

Comment:
HOME is /root for an initialization action.
Should there be a metadata key of a GCS directory to copy here?